source/renderer/app/components/static/About.js

```diff
@@ -76,10 +66,10 @@ export default class About extends Component<Props> {
     const { onOpenExternalLink } = this.props;
     const {
       version, build, os,
-      API, API_VERSION,
+      API_VERSION,
     } = environment;
 
-    const apiName = intl.formatMessage(environmentSpecificMessages[API].apiName);
+    const apiName = intl.formatMessage(environmentSpecificMessages.ada.apiName);
```

Comment:
@daniloprates we no longer need `environmentSpecificMessages` as we are using only ADA now. Please move all those messages under `globalMessages` (which means they will still be in the same file but just in the default export).